.env.example

```
@@ -1,15 +1,7 @@
-REDIS_URL=redis://user:token@ip
-WEBHOOKS_REPLICATOR_TOKEN=
-
-# advanced (optional)
-# These variables are kept for linting compatibility only
-# They should remain empty in a regular usage
-GH_APP_ID=
-GH_APP_INSTALLATION_ID=
+GITHUB_TOKEN=ghp_your-github-token
+GH_APP_ID=0
+GH_APP_INSTALLATION_ID=0
 GH_APP_PRIV_KEY_BASE64=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GH_APP_PRIV_KEY_BASE64 key should go before the GITHUB_TOKEN key

(UnorderedKey)